### ISSUE TITLE
Simplify Not Working Properly

### ISSUE THREAD

**Issue by hypersonicstorm** [nathanfarlow/PineappleCAS#35]
Simplify doesn't seem to be working properly T-T

For example x^2+2x+1 gives x^2+2x+1 instead of (x+1)^2, and 5x+25 gives me 5x+25 instead of 5(x+5).

Expand works fine though, (x+1)^2 turns into x^2+2x+1 and 5(x+5) turns into 5x+25.

Am I doing something wrong? Any help would be appreciated.

**Comment by adriweb** [comment]
Well you're describing factoring but mentioning simplifying?

**Comment by hypersonicstorm** [comment]
idk theres no factoring function in the CAS so I'm assuming its simplify


![image](https://user-images.githubusercontent.com/73567298/193475424-48b37d2b-677e-4d92-9fd0-d7f4e2bb983f.png)
I also saw this somewhere (TIPlanet I think)

**Comment by adriweb** [comment]
Hmm from the screenshots in the readme, I guess it could be. 
if you try things from the screenshots, does it work for you?

it's a bit surprising considering your examples seem simpler...

**Comment by hypersonicstorm** [comment]
Most of them (all of them not including the factoring ones) work

Also I'm pretty sure simplify includes factoring
![image](https://user-images.githubusercontent.com/73567298/193476524-2e465db0-71ec-451f-8d99-318632034849.png)

**Comment by hypersonicstorm** [comment]
I believe this issue was fixed before [here](https://github.com/nathanfarlow/PineappleCAS/issues/3) so I might be using the program wrong

**Comment by nathanfarlow** [comment]
Yep factoring polynomials isn't supported as a simplify operation currently

**Comment by abtblk** [comment]
Is there ever going to be an addition to factor polynomials?
